src/main/kotlin/no/nav/faktureringskomponenten/service/beregning/BeløpBeregner.kt

```diff
@@ -0,0 +1,23 @@
+package no.nav.faktureringskomponenten.service.beregning
+
+import mu.KotlinLogging
+import java.math.BigDecimal
+import java.math.RoundingMode
+import java.time.LocalDate
+
+private val log = KotlinLogging.logger { }
+
+
+class BeløpBeregner {
+    companion object {
+        fun beløpForPeriode(enhetspris: BigDecimal, fom: LocalDate, tom: LocalDate): BigDecimal {
+            val angittAntall = AntallBeregner(fom, tom).beregn()
+            val beløp = enhetspris.multipliserMedHeltall(angittAntall)
+            log.info { "Beløp for periode fom: $fom, tom: $tom regnes med enhetspris $enhetspris og antall: $angittAntall ==> beløp: $beløp" }
```

Comment:
Samme med info vs debug her. Det napper litt i sensitiv-informasjon-følelsen min. 

Kanskje ha samme metode for å formattere tekst overalt? {} vs $